multilabel_classification/requirements.txt

```diff
@@ -0,0 +1,7 @@
+numpy==1.22.0
+scikit-learn==1.0.2
+pillow==9.0.1
+torch==1.12.1+cu113
+timm==0.6.7
+pandas==1.5.0
+torchvision==0.13.1+cu102
```

Comment:
```suggestion
```

Reply:
I don't see where you need torchvision?

Reply:
img = torchvision.io.read_image(path).swapaxes(0, 2).swapaxes(0, 1)

jpeg is not read by PIL/ other libraries, I could add just torch vision without CUDA, 
I'll also see if there are other libraries that can easily display jpeg images

Reply:
This was me trying to use torch vision to download CelebA dataset, I'll keep this resolve open, once I figure this out I'll close it